spec/spec_helper_acceptance.rb

```diff
@@ -5,6 +5,7 @@
 
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
+ENV['BEAKER_FACTER_VAULT_VERSION'] = '1.12.0'
```

Comment:
That file is generated by a template. I took your changes, rebased them and adjusted it for the template: https://github.com/voxpupuli/puppet-vault/pull/55. Thanks a lot for your work!

Reply:
Looking forward for the changes :)